core/arch/arm/kernel/kern.ld.S

```diff
@@ -261,13 +283,24 @@ SECTIONS
 		__start_phys_sdp_mem_section = . ;
 		KEEP(*(phys_sdp_mem_section))
 		__end_phys_sdp_mem_section = . ;
-		. = ALIGN(8);
+
+		/*
+		 * Insure init sections ends on mapping page
```

Comment:
Parts of the code in the init section is even after OP-TEE is fully initialized so it will never be paged out for good.
This aligning wastes memory for with the only win that a few lines of code can be removed from `init_runtime()`